api/workAllocation2/roleService.mock.ts

```diff
@@ -0,0 +1,13 @@
+import MockAdapter from 'axios-mock-adapter';
+import { CASEROLES } from './constants/roles.mock.data';
+
+export const init = (mock: MockAdapter) => {
+  const getCaseRolesUrl = /http:\/\/am-role-assignment-service-aat.service.core-compute-aat.internal\/cases\/[a-fA-F0-9]{16}/;
```

Comment:
Minor query - This probably won't work on preview/demo environments. Does it need to?

Reply:
This is just a mock will be replaced once we have an integration endpoint.

Reply:
ok sure